script.skinshortcuts/addon.xml

```diff
@@ -37,4 +38,8 @@
 		<email></email>
 		<source>https://github.com/BigNoid/script.skinshortcuts</source>
 	</extension>
+	<assets>
+		<icon>icon.png</icon>
+		<fanart>fanart.jpg</fanart>
+	</assets>
```

Comment:
it's in the right spot for sure, though if it was up to me, i'd simply remove it.
it's only supported on krypton and might still log an error on isengard/jarvis.